packages/documentation-site/patternfly-docs/content/developer-resources/dark-theme-handbook.md

```diff
@@ -6,82 +6,26 @@ section: developer-resources
-
-**Black color palette**
-
-Our black color palette was updated and a new, lighter gray was added.
-
-| Variables affected | New or updated color values|
-| -------------------- | -------------------- |
-| --pf-v6-global--palette--black-50  | #e0e0e0 |
-| --pf-v6-global--palette--black-100 | #c6c7c8 |
-| --pf-v6-global--palette--black-200 | #aaabac |
-| --pf-v6-global--palette--black-300 | #868789 |
-| --pf-v6-global--palette--black-400 | #57585a |
-| --pf-v6-global--palette--black-500 | #444548 |
-| --pf-v6-global--palette--black-600 | #36373a |
-| --pf-v6-global--palette--black-700 | #26292d |
-| --pf-v6-global--palette--black-800 | #1b1d21 |
-| --pf-v6-global--palette--black-850 | #212427 |
-| --pf-v6-global--palette--black-900 | #0f1214 |
-| --pf-v6-global--palette--black-1000| #030303 |
-
-**Red color palette**
-
-Two new colors were added to our red color palette.
-
-| Variables affected | New or updated color values|
-| -------------------- | -------------------- |
-| --pf-v6-global--palette--red-9999  | #fe5142 |
-| --pf-v6-global--palette--red-8888  | #ff7468 |
-
-**Blue color palette**
-
-`blue-300` was updated to support an additional variation of primary blue.
-
-| Variable affected | New or updated color values|
-| -------------------- | -------------------- |
-| --pf-v6-global--palette--blue-300 | #1fa7f8 |
-
-
-
-**Global variables**
-
-`--pf-v6-global--BackgroundColor--300` was added as background color for elements that appear on top of others, such as popovers and dropdown menus.
-
-`--pf-v6-global--BackgroundColor--400` was added as a background color for hover and interaction states of items that are placed on top of a dark background, such as dropdown menu items, text inputs, text areas, and menu toggles.
-
-`--pf-v6-global--BorderColor--400` was added as a bottom border color for control elements.
-
-**Status colors**
-
-`--pf-v6-global--primary-color--300` was added as a new primary background color to meet contrast requirements for elements that have text on top, such as primary buttons and unread badges.
-
-**Component style overrides**
-
-Where there are dark theme style overrides needed for specific components, you can find them alongside the components stylesheet in a folder with the structure `themes/dark/[component].scss`. For example, the button component's dark theme overrides are in [components/Button/themes/dark/button.scss](https://github.com/patternfly/patternfly/blob/main/src/patternfly/components/Button/themes/dark/button.scss)
+[Our token package](https://github.com/patternfly/patternfly/tree/v6/src/patternfly/base/tokens) includes files for both light and dark themes. When dark theme is enabled, your product will automatically pull dark theme tokens in order to adapt visual styles appropriately.
```

Comment:
A couple of notes about this. Also just nits, it's fine the way it is!

I _believe_ the word "package" in most of our uses refers to an NPM package, which is a standalone bundle of files that are included in an application. For example, the [react repo has a `/packages` dir](https://github.com/patternfly/patternfly-react/tree/main/packages) and in it are all of the standalone packages we offer that are react based - react-core, react-table, react-charts, etc.

Since the URL points to the v6 branch, we'll want to be sure to update the URL once the v6 code is merged into the main branch since this URL will no longer be valid.

Reply:
ah that makes sense, wdyt about "our token resources include files ...." instead?

Reply:
That sounds good to me! A couple of other ways I might write it in case you like any of these
* our token resources include [styles or stylesheets] for...
* our token stylesheets include [files or styles] for...